synthtool/languages/java.py

```diff
@@ -260,6 +262,7 @@ def gapic_library(
         library=library,
         package_pattern=package_pattern,
         destination_name=destination_name,
+        diregapic=kwargs.get("diregapic", False),
```

Comment:
We can add to the named param list of `gapic_library()` and `bazel_library()` to avoid the `kwargs.get()`